Source/Scene/Model.js

```diff
@@ -322,8 +322,16 @@ define([
 
                 if (gltf instanceof Uint8Array) {
                     // Binary glTF
+                    var result = parseBinaryGltfHeader(gltf);
+
+                    // CESIUM_binary_glTF is from the beginning of the file but
```

Comment:
Where else do you want it mentioned? It is in the doc for the `Model` constructor and the `fromGltf` method.